_data/library/jsfacades.yml

```diff
@@ -126,6 +126,10 @@
       url: https://github.com/fancellu/scalajs-ractive
       desc: Scala.js bindings for [Ractive.js](http://www.ractivejs.org/) and an example application
       dep: ''
+    - name: angulate2
+      url: https://github.com/jokade/angulate2
+      desc: Plugin and bindings for [Angular 2+](http://angular.io) development with Scala.js.
+      dep: 'addSbtPlugin("de.surfice" % "angulate2" % "latest.version")'
```

Comment:
Same about the version number.

Also, I suggest to put this right after `scalajs-angulate` so that Angular-related things are kept together.